doc/chapter-configuration.asciidoc

```diff
@@ -121,10 +121,112 @@ configuration.
 
 === Key Bindings
```

Comment:
Let's follow what we do for remote APIs, and add a note here:

```
_Supported since Newsboat 2.39._
```

This should help users figure out why this doesn't work for their version.

Reply:
Good idea, done